### PR TITLE
Fix "Single Frame" mode not working.

### PR DESCRIPTION
**Pull request by DillyzThe1**
Let's say you have a sprite. One of it's animated pieces use a single symbol with a bunch of various poses and shapes, often variating heavily and relying on single frame mode to allow them to specify a drawing.
You finish your animation and export it as a Texture Atlas, utilizing this library. One problem, though!
All of the symbols you marked as "Single Frame" are now stuck on frame 0, causing you to either have to separate each hand into different symbols (which is bad), or hit break apart on every frame (which is even worse).

This PR simply adds the return for the specified frame to be used in SF mode, allowing for usage of such symbols.

In other words, I'm just allowing you to use a basic Adobe Animate symbol animation technique.

### CODE REVIEW

**Review by MemeHoovy — APPROVED** [review]
nice

**Comment by DillyzThe1** [comment]
i swear i see you everywhere lmao

**Comment by MemeHoovy** [comment]
I could say the same with you lol

**Comment by DillyzThe1** [comment]
we probably just happen to be interested in pretty much the same kind of repos (haxe, fnf, mods, c++, etc) and happen to see each other inside of recommendations

the internet is an interesting place sometimes, but it can also make you want to fry your eyeballs out

**Comment by MemeHoovy** [comment]
yeah XD

**Comment by DillyzThe1** [comment]
context for the last commit: github did the spacing wrong i just downloaded the repo in github desktop and got visual studio code to fix it

**Comment by trayfellow** [comment]
> nice

i also keep seeing u everywhere lol

**Review by TheKitBoi — APPROVED** [review]

**Comment by Cheemsandfriends** [comment]
Oh, I haven't seen this repo.
Sorry 😅
I can't merge this one because it is based on the last version atm.
Altho I think that I accidentally fixed it, I would want to know if this issue still happens on the latest one and I'll merge the patch required 🙂

**Comment by DillyzThe1** [comment]
> Oh, I haven't seen this repo. Sorry 😅 I can't merge this one because it is based on the last version atm. Altho I think that I accidentally fixed it, I would want to know if this issue still happens on the latest one and I'll merge the patch required 🙂

i'll get testing in a moment, but i would heavily recommend just developing on a separate branch next time.
if it fails, i'll update my fork and redo the patch. otherwise, i'll close it.

**Comment by Cheemsandfriends** [comment]
I know you said you would test this but @DillyzThe1 any issue you had regarding this?

**Comment by DillyzThe1** [comment]
whoops, forgot about this pull request
anyway i've had no problem since so there's no need for this to exist
